### ISSUE TITLE
Make Note creation more intuitive

### ISSUE THREAD

**Issue by maxxgl** [wrobbinz/prolificr#13]
The '+' next to the search box can be confusing.

**Comment by wrobbinz** [comment]
Yeah, mainly the fact that I've attached it to the search bar is making it awkward. I did that to quickly get it to play nicely on same line. Also could be an icon like:

<img width="36" alt="screen shot 2018-06-27 at 10 11 55 pm" src="https://user-images.githubusercontent.com/7999878/42014450-40ef929a-7a57-11e8-98a0-8b9e2d5541b9.png">
